configs/restorers/edvr/README.md

```diff
@@ -37,3 +37,15 @@ The metrics are `PSNR / SSIM` .
   year      = {2019},
 }
 ```
+
+## Results and models
```

Comment:
Double `Results and models`.
Please see `https://github.com/open-mmlab/mmediting/blob/master/configs/restorers/edsr/README.md` for the readme format